### ISSUE TITLE
Crash Caused by: java.net.UnknownHostException Unable to resolve host "api.instabug.com": No address associated with hostname 

### ISSUE THREAD

**Issue by SudoPlz** [Instabug/Instabug-React-Native#496]
```
io.reactivex.exceptions.UndeliverableException The exception could not be delivered to the consumer because it has already canceled/disposed the flow or the exception has nowhere to go to begin with. Further reading: https://github.com/ReactiveX/RxJava/wiki/What's-different-in-2.0#error-handling | java.net.UnknownHostException: Unable to resolve host "api.instabug.com": No address associated with hostname 
    RxJavaPlugins.java:367 io.reactivex.plugins.RxJavaPlugins.onError
    CompletableMergeIterable.java:124 io.reactivex.internal.operators.completable.CompletableMergeIterable$MergeCompletableObserver.onError
    CompletableSubscribeOn.java:74 io.reactivex.internal.operators.completable.CompletableSubscribeOn$SubscribeOnObserver.onError
    CompletableAndThenCompletable.java:62 io.reactivex.internal.operators.completable.CompletableAndThenCompletable$SourceObserver.onError
    CompletableAndThenCompletable.java:62 io.reactivex.internal.operators.completable.CompletableAndThenCompletable$SourceObserver.onError
    CompletablePeek.java:95 io.reactivex.internal.operators.completable.CompletablePeek$CompletableObserverImplementation.onError
    ObservableIgnoreElementsCompletable.java:61 io.reactivex.internal.operators.observable.ObservableIgnoreElementsCompletable$IgnoreObservable.onError
    ObservableRetryPredicate.java:90 io.reactivex.internal.operators.observable.ObservableRetryPredicate$RepeatObserver.onError
    ObservableCreate.java:84 io.reactivex.internal.operators.observable.ObservableCreate$CreateEmitter.tryOnError
    ObservableCreate.java:72 io.reactivex.internal.operators.observable.ObservableCreate$CreateEmitter.onError
    NetworkManager.java:104 com.instabug.library.network.NetworkManager$a.subscribe
    ObservableCreate.java:40 io.reactivex.internal.operators.observable.ObservableCreate.subscribeActual
    Observable.java:12284 io.reactivex.Observable.subscribe
    ObservableRetryPredicate.java:112 io.reactivex.internal.operators.observable.ObservableRetryPredicate$RepeatObserver.subscribeNext
    ObservableRetryPredicate.java:41 io.reactivex.internal.operators.observable.ObservableRetryPredicate.subscribeActual
    Observable.java:12284 io.reactivex.Observable.subscribe
    ObservableIgnoreElementsCompletable.java:31 io.reactivex.internal.operators.observable.ObservableIgnoreElementsCompletable.subscribeActual
    Completable.java:2309 io.reactivex.Completable.subscribe
    CompletablePeek.java:51 io.reactivex.internal.operators.completable.CompletablePeek.subscribeActual
    Completable.java:2309 io.reactivex.Completable.subscribe
    CompletableAndThenCompletable.java:35 io.reactivex.internal.operators.completable.CompletableAndThenCompletable.subscribeActual
    Completable.java:2309 io.reactivex.Completable.subscribe
    CompletableAndThenCompletable.java:35 io.reactivex.internal.operators.completable.CompletableAndThenCompletable.subscribeActual
    Completable.java:2309 io.reactivex.Completable.subscribe
    CompletableSubscribeOn.java:64 io.reactivex.internal.operators.completable.CompletableSubscribeOn$SubscribeOnObserver.run
    Scheduler.java:578 io.reactivex.Scheduler$DisposeTask.run
    ScheduledRunnable.java:66 io.reactivex.internal.schedulers.ScheduledRunnable.run
    ScheduledRunnable.java:57 io.reactivex.internal.schedulers.ScheduledRunnable.call
    FutureTask.java:266 java.util.concurrent.FutureTask.run
    ScheduledThreadPoolExecutor.java:301 java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run
    ThreadPoolExecutor.java:1167 java.util.concurrent.ThreadPoolExecutor.runWorker
    ThreadPoolExecutor.java:641 java.util.concurrent.ThreadPoolExecutor$Worker.run
    Thread.java:919 java.lang.Thread.run

Caused by: java.net.UnknownHostException Unable to resolve host "api.instabug.com": No address associated with hostname 
    Inet6AddressImpl.java:124 java.net.Inet6AddressImpl.lookupHostByName
    Inet6AddressImpl.java:103 java.net.Inet6AddressImpl.lookupAllHostAddr
    InetAddress.java:1152 java.net.InetAddress.getAllByName
    Dns.java:41 com.android.okhttp.Dns$1.lookup
    RouteSelector.java:178 com.android.okhttp.internal.http.RouteSelector.resetNextInetSocketAddress
    RouteSelector.java:144 com.android.okhttp.internal.http.RouteSelector.nextProxy
    RouteSelector.java:86 com.android.okhttp.internal.http.RouteSelector.next
    StreamAllocation.java:192 com.android.okhttp.internal.http.StreamAllocation.findConnection
    StreamAllocation.java:144 com.android.okhttp.internal.http.StreamAllocation.findHealthyConnection
    StreamAllocation.java:106 com.android.okhttp.internal.http.StreamAllocation.newStream
    HttpEngine.java:400 com.android.okhttp.internal.http.HttpEngine.connect
    HttpEngine.java:333 com.android.okhttp.internal.http.HttpEngine.sendRequest
    HttpURLConnectionImpl.java:483 com.android.okhttp.internal.huc.HttpURLConnectionImpl.execute
    HttpURLConnectionImpl.java:135 com.android.okhttp.internal.huc.HttpURLConnectionImpl.connect
    HttpURLConnectionImpl.java:266 com.android.okhttp.internal.huc.HttpURLConnectionImpl.getOutputStream
    DelegatingHttpsURLConnection.java:219 com.android.okhttp.internal.huc.DelegatingHttpsURLConnection.getOutputStream
    HttpsURLConnectionImpl.java:30 com.android.okhttp.internal.huc.HttpsURLConnectionImpl.getOutputStream
    NetworkManager.java:8 com.instabug.library.network.NetworkManager.connectWithNormalType
    NetworkManager.java:1 com.instabug.library.network.NetworkManager.access$100
    NetworkManager.java:20 com.instabug.library.network.NetworkManager$a.subscribe
    ObservableCreate.java:40 io.reactivex.internal.operators.observable.ObservableCreate.subscribeActual
    Observable.java:12284 io.reactivex.Observable.subscribe
    ObservableRetryPredicate.java:112 io.reactivex.internal.operators.observable.ObservableRetryPredicate$RepeatObserver.subscribeNext
    ObservableRetryPredicate.java:41 io.reactivex.internal.operators.observable.ObservableRetryPredicate.subscribeActual
    Observable.java:12284 io.reactivex.Observable.subscribe
    ObservableIgnoreElementsCompletable.java:31 io.reactivex.internal.operators.observable.ObservableIgnoreElementsCompletable.subscribeActual
    Completable.java:2309 io.reactivex.Completable.subscribe
    CompletablePeek.java:51 io.reactivex.internal.operators.completable.CompletablePeek.subscribeActual
    Completable.java:2309 io.reactivex.Completable.subscribe
    CompletableAndThenCompletable.java:35 io.reactivex.internal.operators.completable.CompletableAndThenCompletable.subscribeActual
    Completable.java:2309 io.reactivex.Completable.subscribe
    CompletableAndThenCompletable.java:35 io.reactivex.internal.operators.completable.CompletableAndThenCompletable.subscribeActual
    Completable.java:2309 io.reactivex.Completable.subscribe
    CompletableSubscribeOn.java:64 io.reactivex.internal.operators.completable.CompletableSubscribeOn$SubscribeOnObserver.run
    Scheduler.java:578 io.reactivex.Scheduler$DisposeTask.run
    ScheduledRunnable.java:66 io.reactivex.internal.schedulers.ScheduledRunnable.run
    ScheduledRunnable.java:57 io.reactivex.internal.schedulers.ScheduledRunnable.call
    FutureTask.java:266 java.util.concurrent.FutureTask.run
    ScheduledThreadPoolExecutor.java:301 java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run
    ThreadPoolExecutor.java:1167 java.util.concurrent.ThreadPoolExecutor.runWorker
    ThreadPoolExecutor.java:641 java.util.concurrent.ThreadPoolExecutor$Worker.run
    Thread.java:919 java.lang.Thread.run
```

### Expected Behavior
No crashes
### Actual Behavior
Crash

### SDK Version
"instabug-reactnative": {
      "version": "git+https://github.com/Instabug/Instabug-React-Native.git#c28ba0f0e9f77d46df52c79ddc02abe78bd6a904",
      "from": "git+https://github.com/Instabug/Instabug-React-Native.git#customBuild/9.1.6.1"
    },

### React Native, iOS and Android Versions
Android 10, react-native 0.63.2

### Device Model
I don't think it's specific, but this happened to a Samsung S9+

**Comment by AliAbdelfattah** [comment]
@SudoPlz thanks for reporting this, it seems we have a similar issue reported on the Android repo https://github.com/Instabug/Instabug-Android/issues/305. We're working on a fix and we'll update you soon.

**Comment by AliAbdelfattah** [comment]
Hey @SudoPlz, just wanted to give you an update on the issue. The native fix should be included in the next official Android release (estimated within 3 weeks). We'll do a RN release shortly after that.

**Comment by SudoPlz** [comment]
Awesome, thanks for the heads up, looking forward to upgrade, can you please ping me whenever that is out?

I'd be grateful.

Thanks

**Comment by AliAbdelfattah** [comment]
@SudoPlz this has been fixed in v9.1.8.

**Comment by SudoPlz** [comment]
Awesome, thanks just upgraded to it.
